e2e/pvc.go

```diff
@@ -141,7 +154,10 @@ func deletePVCAndPV(c kubernetes.Interface, pvc *v1.PersistentVolumeClaim, pv *v
 	})
 }
 
-func getPVCAndPV(c kubernetes.Interface, pvcName, pvcNamespace string) (*v1.PersistentVolumeClaim, *v1.PersistentVolume, error) {
+func getPVCAndPV(
+	c kubernetes.Interface,
+	pvcName, pvcNamespace string,
```

Comment:
:bell: missed one

Reply:
@nixpanic corrected this too :),, thanks . ptal